ACLPlugin/Source/ACLPlugin/Private/AnimCompress_ACL.cpp

```diff
@@ -183,9 +184,14 @@ void UAnimCompress_ACL::PopulateDDCKey(FArchive& Ar)
 
 	uint32 ForceRebuildVersion = 1;
 	uint16 AlgorithmVersion = get_algorithm_version(AlgorithmType8::UniformlySampled);
-	uint32 SettingsHash = Settings.get_hash();
```

Comment:
Looks like the settings hash was removed by mistake

Reply:
Sorry :(